test/Microsoft.DotNet.MSBuildSdkResolver.Tests/GivenAnMSBuildSdkResolver.cs

```diff
@@ -274,7 +278,9 @@ public void ItReturnsHighestSdkAvailableThatIsCompatibleWithMSBuildWhenVersionIn
             result.Success.Should().BeTrue($"No error expected. Error encountered: {string.Join(Environment.NewLine, result.Errors ?? new string[] { })}. Mocked Process Path: {environment.ProcessPath}. Mocked Path: {environment.PathEnvironmentVariable}");
             result.Path.Should().Be((disallowPreviews ? compatibleRtm : compatiblePreview).FullName);
             result.AdditionalPaths.Should().BeNull();
-            result.PropertiesToAdd.Count.Should().Be(2);
+            result.PropertiesToAdd.Count.Should().Be(4);
+            result.PropertiesToAdd.Should().ContainKey(_dotnetHost);
+            result.PropertiesToAdd.Should().ContainKey(_msbuildNetTaskHostRuntimeVersion);
             result.PropertiesToAdd.ContainsKey("SdkResolverHonoredGlobalJson");
             result.PropertiesToAdd.ContainsKey("SdkResolverGlobalJsonPath");
```

Comment:
What's the difference between `.Should().ContainKey()` and `.ContainsKey()`?

Reply:
Interesting. `ContainsKey` just returns bool. And then there is no assertion. It was probably meant to be `Shgould().ContainKey()` instead. I will fix this